dotfiles/prompt

```diff
@@ -160,3 +160,6 @@ _prompt() {
 }
 
 PROMPT_COMMAND='_prompt'
+if [[ "$TERM_PROGRAM" = Apple_Terminal ]]; then
+  export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
```

Comment:
Okay. Wow. Reading fail. Yes that looks good 😅



> On Jun 6, 2015, at 11:21 PM, Will Strinz <notifications@github.com> wrote:
> 
> In dotfiles/prompt:
> 
> > @@ -160,3 +160,6 @@ _prompt() {
> >  }
> >  
> >  PROMPT_COMMAND='_prompt'
> > +if [[ "$TERM_PROGRAM" = Apple_Terminal ]]; then
> > +  export PROMPT_COMMAND="$PROMPT_COMMAND;update_terminal_cwd;"
> I don't think so; that's why I added L163, since weirdly iTerm doesn't have the command. So it should only trigger if you're using the Apple Terminal.
> 
> —
> Reply to this email directly or view it on GitHub.
>